packages/server/lib/gui/auth.js

```diff
@@ -191,8 +191,8 @@ const _launchNativeAuth = (loginUrl, sendMessage) => {
   openExternalAttempted = true
 
   // wrap openExternal here in case `electron.shell` is not available (during tests)
-  return Promise.fromCallback((cb) => {
-    shell.openExternal(loginUrl, {}, cb)
+  return Promise.try(() => {
+    return shell.openExternal(loginUrl)
```

Comment:
`shell.openExternal` is promisified right? Not necessary to do `Promise.try` anymore

Reply:
yeah, it's promisified. still using promise.try since `shell.openExternal` doesn't exist in tests, but i'll change it to use proper stubbing.